spec/exporters/conference_preferences_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+RSpec.describe Exporters::ConferencePreferences do
+
+  describe '#current' do
+    let!(:old_team) { create :team, :with_preferences, name: "OLDTEAM" }
+    let!(:new_team) { create :team, :with_preferences, :in_current_season, name: "NEWTEAM" }
+
+    subject { described_class.current }
+
+    it 'exports conference preferences for teams of the current season' do
```

Comment:
Hm, I feel like this is under the hood just testing the `ConferencePreference`'s `current_team` scope, not actually the exporter in itself.

Can you come up with a test that better targets the exporter?